src/devices/bus/coco/coco_multi.cpp

```diff
@@ -233,7 +264,19 @@ void coco_multipak_device::device_start()
 
 void coco_multipak_device::device_reset()
 {
-	m_select = 0xFF;
+	set_select(MULTI_SLOT_LOOKUP[ioport(SWITCH_CONFIG_TAG)->read()]);
+	m_block = 0;
+}
+
+
+//-------------------------------------------------
+//  switch_changed - panel switch changed
+//-------------------------------------------------
+
+INPUT_CHANGED_MEMBER( coco_multipak_device::switch_changed )
+{
+	if( m_block == 0 )
```

Comment:
The spacing around this doesn't match the other `if` statements in the file - please be consistent with surrounding code.  The rest of the file is `if (condition)` style.